### DISCUSSION TITLE
v3.4.5 Fix - Earthquake

### DISCUSSION THREAD

**Discussion by byRoadrunner** [CloudNetService/CloudNet-v3#690] [Releases]
# Update v3.4.5 - Earthquake #

**Fixes:**
* Fixed exception on ServerListPingEvent in 1.19

**Changes:**
* Updated the backup versions to 1.19

**All Commits**: https://github.com/CloudNetService/CloudNet-v3/compare/v3.4.4...v3.4.5

<hr /><em>This discussion was created from the release <a href='https://github.com/CloudNetService/CloudNet-v3/releases/tag/v3.4.5'>v3.4.5 Fix - Earthquake </a>.</em>

**Comment by ghost** [comment]
Hello!

**Comment by 011248163264** [comment]
hi is this version also running with java 17? or only with java11
